### PR TITLE
PR: Bugfixes after 0.1

### LINKED ISSUE #97: Add setup.cfg to create universal wheels
Example: https://github.com/spyder-ide/spyder-unittest/blob/master/setup.cfg

### LINKED ISSUE #98: Change Spyder version from setup.py
Delete dev0 from the Spyder version.

### LINKED ISSUE #99: Include runtests.py in MANIFEST.in

### LINKED ISSUE #101: Files in MANIFEST.in not included in the installation
Add to `setup.py`  > `include_package_data=True`

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #97 
Fixes #98 
Fixes #99 
Fixes #101

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def get_version(module='spyder_notebook'):
                      "to run inside Spyder as a plugin.",
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=REQUIREMENTS,
+    include_package_data=True,
```
Comment (ccordoba12) [inline setup.py]: Did you check that after adding this option our wheels include all data files needed by the package?

Comment (ccordoba12) [inline setup.py]: Probably you don't know, but wheels are zip files. So you can easily inspect its contents :-)

Comment (dalthviz) [inline setup.py]: Yep, tested installing with the wheel generated by `python setup.py bdist_wheel`

Comment (ccordoba12) [inline setup.py]: Great!

**Pull request merged**